src/helpers/similarity.ts

```diff
@@ -0,0 +1,86 @@
+import { getBotContext } from "../bindings";
+import { Payload, Choices } from "../types";
+import { listAllIssuesForRepo, upsertCommentToIssue } from "../helpers/issue";
+import axios from "axios";
+
+export const findDuplicateOne = async () => {
+  const { payload: _payload } = getBotContext();
+  const issue = (_payload as Payload).issue;
+  const issues = await listAllIssuesForRepo("all");
```

Comment:
RFC @pavlovcik

Reply:
In that case `listAllIssuesForRepo` should be fixed to filter out PRs because the name implies only issues.

Currently we already store issues as a part of analytics so we can change to store it when the issue is opened and add description to the schema. Then we can query all of the issues in an efficient manner.